### DISCUSSION TITLE
Issue Policy

### DISCUSSION THREAD

**Discussion by dmb2** [stumpwm/stumpwm#1169] [Announcements]
StumpWM issues fall broadly into these categories:
1. Legitimate bugs that can be reproduced by those of us with commit access
2. Bugs that cannot be reproduced, and for which the submitter is unresponsive
3. Support on aspects of lisp/api/confusion about how a bit of code works

Support issues that do not involved confirmed bugs should be submitted as discussions. 

Bugs that can be reproduced will be kept open.

Bugs that cannot be reproduced which have had no response will be closed as unplanned.

**Comment by Hexstream** [comment]
> Bugs that cannot be reproduced, and for which the submitter is unresponsive

I think `and` should be `or` or `and/or`.

([I don't think I was "unresponsive"](https://github.com/stumpwm/stumpwm/issues/649), but inability to reproduce should be enough.)
